### PR TITLE
Fix Part of #4: Created `Mechanical Department` Page 

### PR DESCRIPTION
**Pull request by jeakansh**
## Explanation

Fixes Part of #4 : Created `Mechanical Department` Page which is a part of Department Page.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR is made from a branch that's **not** called "master".
- [x] The PR is made from a branch that is up-to-date with "master".
- [x] The PR's branch is based on "master" and not on any other branch.

### CODE REVIEW

**Review by jkugsiya — CHANGES_REQUESTED** [review]
as you can see below image that the content is not properly responsive (the navbar-menu icon is hidden and some text is getting hidden) and the user needs to swipe the screen in order to see the full content, that is not the expected behaviour.
image

**Comment by jeakansh** [comment]
Responsive
The nav-bar and the complete content is Responsive.

**Review by jkugsiya — CHANGES_REQUESTED** [review]
developer options - iPad orientation portrait. Look at the navbar and also the information cards they are going out of screen bounds and hiding the written content in it:

image

Also why this blur affect on the footer:

image

File: mechanical_Department.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <br>
+        <h3 class="h3-left-border">Our Faculty</h3>
+        <div class="our-faculty-mech ">
+            <div class="row justify-content-center">
+                <div class= "card col-lg-3 col-md-4 col-10 col-sm-3 mech-faculty-card card-no-shadow" >
```
Comment (jkugsiya) [inline mechanical_Department.html:189]: This one is col-md-4 and others are col-md-3. Make this too col-md-3

Comment (jkugsiya) [reply mechanical_Department.html:189]: Also use col-sm-4 everywhere in the faculty images card

Comment (jeakansh) [reply mechanical_Department.html]: I have done all the required Changes.

**Review by jkugsiya — APPROVED** [review]
LGTM, Thanks! @jeakansh

**Pull request merged**